File: Java/UberVan.java
package Java;

public class UberVan extends Car {
  String brand;
  String model;

  public UberVan(String license, Account driver, String brand, String model) {
    super(license, driver);
    this.brand = brand;
    this.model = model;
  }

  public String getBrand() {
    return brand;
  }

  public void setBrand(String brand) {
    this.brand = brand;
  }

  public String getModel() {
    return model;
  }

  public void setModel(String model) {
    this.model = model;
  }

  @Override
  public void setPassenger(Integer passenger) {
    if (passenger == 6) {
      super.setPassenger(passenger);
    } else {
      System.out.println("Debe tener 6 asientos");
    }
  }

}

File: PHP/UberVan.php
<?php

require_once('Car.php');

class UberX extends Car{ 
    public $brand;
    public $model;

    function __construct($license, $driver, $passenger, $brand, $model) {
      parent::__construct($license,$driver,$passenger);
      $this->brand = $brand;
      $this->model = $model;
  }

  public function setPassenger($passenger) {

    if (passenger == 6) {
      this.passenger = passenger;
    } else {
      System.out.println("debe tener 6 asientos");
    }
  }
}
